jscomp/bsb/bsb_merlin_gen.ml

```diff
@@ -86,18 +98,27 @@ let output_merlin_namespace buffer ns=
     Buffer.add_string buffer "-open ";
     Buffer.add_string buffer x 
 
+#if BS_NATIVE then
+let bsc_flg_to_merlin_ocamlc_flg bsc_flags backend  =
+#else
```

Comment:
could `backend` be made a lazy value so that the patch does not need be intrusive (the argument does not need be passed across each function), so is `nest` argument

```
let backend = lazy (...)
```